docs/markdown/Builtin-options.md

```diff
@@ -134,6 +134,7 @@ available on all platforms or with all compilers:
 | b_ndebug      | false          | true, false, if-release                                          | Disable asserts                                                               |
 | b_pch         | true           | true, false                                                      | Use precompiled headers                                                       |
 | b_pgo         | off            | off, generate, use                                               | Use profile guided optimization                                               |
+| b_profile     | off            | off, prof, gprof                                                 | Generate extra code to write profile information                              |
```

Comment:
Should I get rid of the 'prof' option and just make this a Boolean option (gprof|True/False)? I couldn't find 'prof' available in any Debian or Arch packages. I couldn't find docs online for it either. Any search I tried only returned results for using 'gprof'.

I only thought to add 'prof' because it was mentioned in the gcc docs:

>Generate extra code to write profile information suitable for the analysis program prof (for -p) or gprof (for -pg).

Reply:
I think prof only in theory exists on proprietary systems that shipped their own, but the GNU version more or less won everywhere.